encrypt-image/install-skopeo.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -euo pipefail
+# Check if the DEBUG env var is set to true
+if [ "${DEBUG:-false}" = "true" ]; then
+    set -x
+fi
+
+# Check if skopeo is already installed then skip this script
+if command -v skopeo >/dev/null 2>&1; then
+    echo "skopeo is already installed, skipping..."
```

Comment:
We should verify the installed skopeo version. I have encountered issues where the installed one in the default path was older and didn't support encryption

Reply:
done.